develop/integrations/wallets.md

```diff
@@ -1,4 +1,60 @@
 ---
 title: Wallets
-description: TODO
----
\ No newline at end of file
+description: Explore blockchain wallets: securely manage digital assets with hot wallets for online access or cold wallets for offline, enhanced security.
+---
+
+# Wallets
+
+## What is a Blockchain Wallet?
+
+A wallet serves as your gateway to interacting with blockchain networks. Rather than storing funds, wallets secure your private keys, controlling access to your blockchain assets. Your private key provides complete control over all permitted transactions on your blockchain account, making it essential to keep it secure.
+
+Wallet types fall into two categories based on their connection to the internet:
+
+- [**Hot Wallets**](#hot-wallets) - online storage through websites, browser extensions or smartphone apps
+- [**Cold Wallets**](#cold-wallets) - offline storage using hardware devices or air-gapped systems
+
+## Hot Wallets
+
+<div class="grid cards" markdown>
+
+-   :material-clock-fast:{ .lg .middle } __Talisman__
+
+    ---
+
+    A better way to explore Web3. Keep your assets safe, manage your portfolio, and explore Polkadot and Ethereum apps with Talisman. Interact with Web3 apps, store your favorite crypto assets, and manage your accounts on over 150+ Substrate and EVM networks. NFT Management, ledger Support, fiat On-ramp, portfolio tracking.
```

Comment:
Again, trying to avoid a marketing writing style here:

```suggestion
    With Talisman, you can securely store assets, manage your portfolio, and interact with Polkadot and Ethereum applications. It supports Web3 apps, asset storage, and account management across over 150 Polkadot SDK-based and EVM networks. Additional features include NFT management, Ledger support, fiat on-ramp, and portfolio tracking
```